pkgs/applications/misc/dozenal/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchFromGitHub, ncurses, hdate, lua5_2 }:
+
+stdenv.mkDerivation rec {
+  name = "dozenal";
+  version = "v12010904";
```

Comment:
Note that `version` should be a part of `name` and should not contain the `v` prefix.  You can check `builtins.parseDrvName` to see how version strings are parsed out of the derivation name.